### DISCUSSION TITLE
How to hide year view?

### DISCUSSION THREAD

**Discussion by dusanmanic** [wojtekmaj/react-calendar#609] [Q&A] [answered]
Hello to all. I have a question according to react calendar.  I was looking but I didn`t find the answer, is there an option to hide the year. So the only options are to be month and days. Thanks in advance.

**Comment by wojtekmaj** [comment, marked as answer]
You're looking for `minDetail` prop. See README for details.

**Reply by dusanmanic** [reply]
Thank you for the fast answer, I was asking more for a year in the title. For example, write only "March" not "March 2022".

**Reply by wojtekmaj** [reply]
You could write a function that returns only month and pass it to `formatMonthYear`, it's currently only used in the top navigation, but it's not guaranteed it won't be ever used in other contexts.

**Reply by dusanmanic** [reply]
It is working, I really appreciate your help.
